llvm/tools/llvm-split/llvm-split.cpp

```diff
@@ -123,6 +189,16 @@ int main(int argc, char **argv) {
     Out->keep();
   };
 
+  if (SYCLSplitMode != IRSplitMode::IRSM_NONE) {
+    auto E = runSYCLSplitModule(std::move(M));
+    if (E) {
+      errs() << E << "\n";
```

Comment:
Should we return 1 if there's an error? Looks like there's a chance we return 0 on error here.